hypothesis-python/tests/nocover/test_simple_numbers.py

```diff
@@ -93,7 +93,7 @@ def f(x):
 
 def test_minimal_non_boundary_float():
     x = minimal(floats(min_value=1, max_value=9), lambda x: x > 2)
-    assert 2 < x < 3
+    assert x == 3  # (the smallest integer > 2)
```

Comment:
This is good, and I'll want to additionally tighten up the other tests to check we get the exact shrinking behavior we want.  *After* we deal with the large-negative-exponent thing.

Reply:
Did some tightening, in this file at least. Any other places?

Reply:
This file is all that springs to mind, and it looks like you've tightened it all up 👍 

Oh, how about adding a new test that floats between say (1.1, 1.6) shrinks to 1.5?  Asymmetric interval going to nice values, rather than nice fractions of the allowed interval.